ontology/yaml/resources/units/units.yaml

```diff
@@ -61,45 +130,145 @@ linearacceleration:
 - millimeters_per_square_second
 linearvelocity:
 - meters_per_second: STANDARD
+- meters_per_minute
+- meters_per_hour
+- millimeters_per_minute
+- millimeters_per_second
+- kilometers_per_hour
 - feet_per_minute
+- feet_per_second
 - miles_per_hour
+luminance:
+- candelas_per_square_meter: STANDARD
+luminousflux:
+- lumens: STANDARD
+luminousintensity:
+- candelas: STANDARD
+magneticflux:
+- webers: STANDARD
+magneticinduction:
+- teslas: STANDARD
+magneticmoment:
+- ampere_square_meters: STANDARD
+mass:
+- grams: STANDARD
+- kilograms
+- pounds_mass
+- tons
+massflow:
+- grams_per_second: STANDARD
+- grams_per_minute
+- kilograms_per_hour
+- kilograms_per_minute
+- kilograms_per_second
+- tons_per_hour
+- pounds_mass_per_hour
+- pounds_mass_per_minute
+- pounds_mass_per_second
+obscuration:
+- percent_obscuration_per_meter: STANDARD
+- percent_obscuration_per_foot
 percentage:
 - percent: STANDARD
-powerfactor:
-- no_units: STANDARD
 power:
 - watts: STANDARD
 - kilowatts
-- tons_of_refrigeration
+- volt_amperes
 - kilovolt_amperes
-- kilovolt_amperes_reactive
 - megawatts
+- milliwatts
+- tons_of_refrigeration
+- btus_per_hour
+- horsepower
+- kilobtus_per_hour
+powerfactor:
+- no_units: STANDARD
+powerflux:
+- watts_per_square_meter: STANDARD
+- watts_per_square_foot
 pressure:
 - pascals: STANDARD
-- bars
+- hectopascals
 - kilopascals
+- centimeters_of_mercury
+- centimeters_of_water
+- millimeters_of_mercury
 - inches_of_water
+- inches_of_mercury
 - pounds_force_per_square_inch
-- atmospheres
+- bars
+- millibars
+pressurechange:
```

Comment:
what is this? not sure what this is.

Reply:
It is the unit [**atm**](https://en.wikipedia.org/wiki/Standard_atmosphere_(unit))

Reply:
I had actually removed it, but I can add it back.

Reply:
i am confused. where is this used? what is this fundamental concept? i am not familiar with anything that is derived as pressure per unit temperature.

Reply:
`atmosphere` is a pressure unit, what is confusing?

Reply:
So it is not about `atmosphere` and rather about `psi_per_degree_farenheit`?
If so, this comes directly from the BACnet standard. See [this example](https://store.chipkin.com/articles/bacnet-the-loop-object) of where it is used to define a PID loop.

Reply:
No my question is more around what the phenomenon is we are measuring. For example, `power` is the rate of change of energy into a system; it is in units of force-distance over unit time. Almost all of the concepts you have defined here are named after the phenomenon they describe. This is the exception. 

It sounds like the application is for a PID controller? I assume it takes as input degrees fahrenheit and outputs PSI, and so this is simply an arbitrary conversion factor? 

If that is the case, I am having a hard time understanding the use case considering we do not model PID loops (they are way too low level). Is this really something we want to commit?

Reply:
@pisuke nudge for the above conversation :)